actions.go

```diff
@@ -83,7 +83,11 @@ func PipelineTemplatePublishAction(clientConfig spinnaker.ClientConfig) cli.Acti
 		}
 
 		logrus.Info("Publishing template")
-		ref, err := client.PublishTemplate(template, cc.Bool("skipPlan"))
+		options := make(map[string]interface{})
```

Comment:
I think I would rather this be a struct. One of the major downfalls of Spinnaker's internals is how everything is an unbounded map - would like to avoid that in roer wherever possible.

```go
type publishTemplateOptions struct {
  SkipPlan bool
  TemplateID string
  Source string
}

// ...

ref, err := client.PublishTemplate(template, publishTemplateOptions{
  SkipPlan: cc.Bool("skipPlan"),
  TemplateID: cc.String("templateId"),
  Source: cc.String("source"),
})
```